cmd/clusterctl/client/repository/metadata_client.go

```diff
@@ -84,6 +84,9 @@ func (f *metadataClient) Get(ctx context.Context) (*clusterctlv1.Metadata, error
 		log.V(1).Info("Using", "override", metadataFile, "provider", f.provider.ManifestLabel(), "version", version)
 	}
 
+	// Dump the metadata.yaml content to logs for debugging
+	log.V(5).Info("metadata.yaml content", "provider", f.provider.ManifestLabel(), "version", f.version, "content", string(file))
```

Comment:
I think we should not log config from the metadata file (generally we are not logging content from config files)

Looking at this from another angle, those kind of errors will mostrly hit users setting up a provider repository, and thus they already know the content of the file.

If those errors hits users, and the provider repository is not set up properly, there is not much they can do even if the log reports the full content of the file

Reply:
That's right. I deleted it.